data.sql

```diff
@@ -55,3 +54,149 @@ WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
 UPDATE animals
 SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
 WHERE name = 'Angemon'  OR name = 'Boarmon';
+
+/* Vets*/
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'William Tatcher', 45, '23/04/2000');
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'Maisy Smith', 26, '17/01/2019');
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'Stephaine Mendez', 64, '04/05/1981');
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'Jack Harkness', 38, '08/06/2008');
+
+/* Specialisations */
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Pokemon'),
+  (SELECT id FROM vets WHERE name = 'William Tatcher')
+);
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Digimon'),
+  (SELECT id FROM vets WHERE name = 'Stephaine Mendez')
+);
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Digimon'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness')
+);
+
```

Comment:
- [ ] You missed adding the following data to `specialization`, Please make sure to add that.
  - Vet Stephanie Mendez is specialized in Pokemon.